### ISSUE TITLE
How to integrate rocksplicator with exisiting application running on rocksdb.

### ISSUE THREAD

**Issue by anip30** [pinterest/rocksplicator#154]
I have a standalone application running on rocksdb. I want to make it distributed for hackathon. I don't want to replace rocksdb with any distributed DB so we decided to use rocksplicator. 

Do we need to change our application to use rocksdplicator or it is separate process. We just need to point it to the DBs.

**Comment by newpoo** [comment]
It's a library, so you need to compile the library into your application process.

You need to open rocksdb instances in your application code, and add opened rocksdb instance to rocksplicator library at both Master and Slave sides. The library will do the data replication for you.

**Comment by anip30** [comment]
you mean only rocksdb_replicator library not rocksdb_admin. 

You mean opening the DB and all stay as it is but we need to add extra code to add that DB handle for replication.

**Comment by newpoo** [comment]
Yes, opening db is still done by your application, but you need to add code to add db handlers to rocksplicator library. Also, you will need to write/read to/from db through the handlers returned from rocksplicator.
